index.html

```diff
@@ -2905,6 +2915,152 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls one or more
```

Comment:
isn't it the holder who controls the confirmation methods?

```suggestion
credential</a> to declare that the <a>holder</a> controls one or more
```

Reply:
@Sakurann I think that should be included now. Can you please check again and does the new language work for you?